AzureData/README.md

```diff
@@ -605,22 +605,38 @@ database.get (userWithId: userId) { r in
 
 #### Delete
 ```swift
-AzureData.delete (user, fromDatabase: databaseId) { s in
-    // s == successfully deleted
+AzureData.delete (user, fromDatabase: database) { r in
```

Comment:
this override should just be `AzureData.delete(user)`.  we don't need the database if we have the user object as we can just call the generic `AzureData.delete(Resource)`